protocols/deflate/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "libp2p-deflate"
+edition = "2018"
+description = "Deflate encryption protocol for libp2p"
+version = "0.8.1"
```

Comment:
Oh, also, this should be `0.1.0`.

Reply:
@tomaka Good catch! I actually thought it should be 0.8.1 since it looked as if the other protocols followed the version of the core crate. Should have asked you, but didn't think of it.